rust/src/nfs/nfs3_records.rs

```diff
@@ -301,36 +301,36 @@ pub(crate) fn many0_nfs3_response_readdirplus_entries<'a>(
     many0(complete(parse_nfs3_response_readdirplus_entry_cond))(input)
 }
 
+
 #[derive(Debug, PartialEq)]
 pub struct Nfs3RequestReaddirplus<'a> {
     pub handle: Nfs3Handle<'a>,
-
-    pub cookie: u32,
+    //pub cookie: u32,
+    pub cookie: u64, // cookie is a u64 size
     pub verifier: &'a [u8],
     pub dircount: u32,
-    pub maxcount: u32,
+    //pub maxcount: u32,
 }
```

Comment:
`cookie` field at `READDIR Call` is of 8byte size, and no `maxcount` field.

I left the original code as commented for reviewing.

Reply:
I see maxcount in my pcaps, why do you believe the maxcount is incorrect?

Reply:
never mind those, that was when I assumed that the `**plus` record type do not exist, and I left the comments etc, as a draft, my bad!